KifuSF/DetailedDonationModularTableViewController.swift

```diff
@@ -42,7 +42,26 @@ class DetailedDonationModularTableViewController: ModularTableViewController {
 
     // MARK: - Methods
     @objc func flagButtonPressed() {
-        //TODO: alex-flagging
+        let flaggingViewController = UINavigationController(
+            rootViewController: FlaggingViewController(
+                flaggableItems: flaggableItems,
+                donationToReport: donation
+            )
+        )
+
+        let alertController = UIAlertController(
+            title: "Is there anything wrong with this donation?",
+            message: nil,
+            preferredStyle: .actionSheet
+        )
+
+        alertController.addButton(
+            title: "Report Donation",
+            style: .default) { (_) in
+                self.present(flaggingViewController, animated: true)
+        }
```

Comment:
are we only reporting a donation here? if so, there is no need to pass in a `user` in the constructor of the FlaggingVc

Reply:
Agree! I wasn't sure how the FlaggingService worked, so I had the urge to use both constructor parameters :]